### PR TITLE
Update deploy-apps.md

### PR DESCRIPTION
**Pull request by ElaineAnsell**
Updates to Store for Business purchase apps flow

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/get-started/deploy-apps.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ If you haven't signed up with Microsoft Store for Business, you can sign up when
 1. Sign in to [Microsoft Store for Business](https://businessstore.microsoft.com) with your Microsoft Store for Business Admin account.
 2. Select **Shop for my group**.
 3. Use Search to find that the app that you want, and select the app.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/deploy-apps.md]:
```suggestion
3. Use Search to find the app that you want, and select the app.
```

File: microsoft-365/managed-desktop/get-started/deploy-apps.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ If you haven't signed up with Microsoft Store for Business, you can sign up when
...
-4. On the product details, select **Get the App**. 
+4. On the product details, select **Get the App**.  Click Manage, then select Private store availability to choose which groups of users in your organization this app can use this app.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/deploy-apps.md:56]:
```suggestion
4. On the product details, select **Get the App**.  Choose **Manage**, then select **Private store availability** to choose which groups of users in your organization can use this app.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - one additional edit as well, please.

**Comment by yogkumgit** [comment]
@ElaineAnsell Please implement the suggested changes. Thanks

**Comment by ElaineAnsell** [comment]
Edit looks good.

**Pull request closed without merging**